controllers/CodeExecutorController.js

```diff
@@ -0,0 +1,49 @@
+const { validateWithTests } = require('../schemas/requestSchemas')
```

Comment:
Ogólnie z tego co rozumiem to w nodzie odpowiednikiem Controllera czyli rozdzielania dla danego prefiksu endpointa jest router, w którym zgodnie z tym co mamy w RecruitmentService nie powinno być logiki, a do logiki bardziej pasuje mi service, więc jakoś ujednolicił/zmieniłbym nazwę